engine/engine.go

```diff
@@ -97,6 +109,44 @@ func (e *Engine) Run(ctx context.Context, step *spec.Step, output io.Writer) (*r
 	return exec.Run(ctx, step, output)
 }
 
+func fileFolderCreation(paths []*spec.File) error {
+	for _, f := range paths {
+		if f.Path != "" {
```

Comment:
```
if f.Path == "" {
  continue
}

path := f.Path
...
```

This will reduce one nested block and make code easier to read.

Reply:
Agreed. I'll fix this because TP's on the vacation.